### ISSUE TITLE
Perisylvian hypermetabolism in FDG PET

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7147]
**Preferred term label:**
Perisylvian hypermetabolism in FDG PET

**Synonyms**

**Definition (free text, please give PubMed ID)**
todo

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
300643
PMID: 16497722. 

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Adding
Def:
18F‐fluorodeoxyglucose (FDG) positron emission tomography and computed tomography (FDG‐PET/CT) evaluates the glucose metabolism of the brain. FDG uptake is interpreted to represent glycolysis. This term therefore represents an abnormally increased FDG uptake (increased glucose metabolism) in the region of the brain that surrounds the Sylvian fissure.
PMID: 33452041
